airflow-core/src/airflow/plugins_manager.py

```diff
@@ -507,7 +507,7 @@ def integrate_macros_plugins() -> None:
     global plugins
     global macros_modules
 
-    from airflow import macros
+    from airflow.sdk.definitions import macros
```

Comment:
I'm not so sure about this -- can we do the macro integration only in the SDK side somehow? The reason is I'd like us to avoid adding new dependencies between core and task sdk

Reply:
Yeah thats fair. Let me see what i can do.

Although the registration with plugins manager which is in the core would create an implicit dependency right?